apps/www/app/(main)/door/[code]/(public)/@navbar/_client/AddContact.tsx

```diff
@@ -35,14 +41,14 @@ export default function AddContact({ profile_id }: { profile_id: string }) {
 
   return match([toggle_contact, find_contact])
     .with([{ status: "loading" }, P._], [P._, { status: "loading" }], () => (
-      <ActionItem>
+      <div>
         <Spinner className="size-5" />
-      </ActionItem>
+      </div>
     ))
     .otherwise(() => (
-      <ActionItem
-        className="text-left"
-        onAction={toggle_add_contact}
+      <Button
+        className={`${className} flex gap-2 text-left`}
```

Comment:
Kind of dangerous as tailwind class order matter. I would recommend using a local tv at least.

  ```suggestion
        className={tv("flex gap-2 text-left")({className})}
```